implementations/dotnet-corvus-jsonschema/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
```

Comment:
The build segfault is because you're missing a `--platform=$BUILDPLATFORM` (see also the other dockerfile).

Reply:
I can't see anything in the repo with `--platform`?

Reply:
https://github.com/bowtie-json-schema/bowtie/blob/main/implementations/dotnet-jsonschema-net/Dockerfile#L1

Reply:
Ah! I wasn't synced with that version.

Reply:
(Let me revert all of that mess.)